javascript-data-model/javascript-data-model/main.js

```diff
@@ -2,27 +2,31 @@
 
 var orderHistory = [
   {
-    customerName: 'John Doe',
-    city: 'Los Angeles',
-    items: 'Bose Headphones',
-    freeGift: true
+    orderNo: 11439416898772232,
+    dateOfOrder: 'Aug 4 2020',
+    customerName: 'JS Masher',
+    items: 'JavaScript for impatient programmers',
```

Comment:
Look at the output for each item. They include the item title, cost, and an image of the item. Is a string description of the item name going to be enough data to do that?